app/models/task.py

```diff
@@ -3,3 +3,33 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String, nullable=False)
```

Comment:
👍 Great job using `nullable=False` for an extra layer of defense between a client's potentially bad form data and your database!